src/ContaoManager/Plugin.php

```diff
@@ -14,8 +17,10 @@ class Plugin implements BundlePluginInterface
     public function getBundles(ParserInterface $parser)
     {
         return [
-            BundleConfig::create('HeimrichHannot\VersionsBundle\VersionsBundle')
-                ->setLoadAfter(['Contao\CoreBundle\ContaoCoreBundle'])
+            BundleConfig::create(VersionsBundle::class)
```

Comment:
2 bundles are created? Correct?

Reply:
Yeah, VersionsBundle ist the current (legacy) bundle class, the other one is the new one with correct conventions. I kepts the old class cause of bc. We already have such a setup in utils bundle. The next major release will remove this duplicate.

Reply:
Ah, I suspected that. Thanks for clearification.